packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -64,6 +65,17 @@ export namespace WorkspaceCommands {
         id: 'file.compare',
         label: 'Compare with Each Other'
     };
+    export const ADD_FOLDER: Command = {
+        id: 'workspace:addFolder',
+        label: 'Add Folder to Workspace...'
+    };
+    export const REMOVE_FOLDER: Command = {
+        id: 'workspace:removeFolder'
+    };
+    export const OPEN_ROOT_CONFIG: Command = {
+        id: 'workspace:openRootConfig',
+        label: 'Open Workspace Root Config File'
```

Comment:
Why _Workspace **Root** Config File_ and not _Workspace Config File_?

Reply:
in my opinion the **workspace config** consists of the "roots.config" & "settings.json" at least. Probably "tasks.json" should be taken into account too. that's why i am calling it "root config" and not just "config". feel free to disagree if you think otherwise